app/libs/classes/viva_application.py

```diff
@@ -95,42 +95,33 @@ def _create_new_application(self):
         if not client:
             raise ValueError(f'Client can not be {client}. Verify your tags!')
 
+        attachment_list = self._get_completion_attachments()
```

Comment:
We can probably remove `_completion` from the name of every attachment function now that attachments are not strictly connected to a completion.